docs/source/models/causes/maternal_disorders/gbd_2021_mncnh/postpartum_depression.rst

```diff
@@ -0,0 +1,275 @@
+
+We will conservatively assume that postpartum depression is not a cause/contributor of maternal mortality due to suicide or other causes in our simulation. While the GBD estimates deaths due to self-harm, these estimates are not specific to the postpartum population and therefore may not be generalizable.  
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
++++++
+
+We will not model PPD as a dynamic transition model, but rather a probabilistic condition that begins at the time of birth and persists for some specified duration. The probability of experiencing PPD will be informed by a ratio per birth from the literature. PPD will be modeled as a YLD-only cause, although this is a limitation of our model as PPD may be associated with risk of suicide or infanticide in extreme cases.
+
+Restrictions
+++++++++++++
+
+The following table describes any restrictions in the
+effects of this cause (such as being only fatal or only nonfatal), as
+well as restrictions on the ages and sexes to which the cause applies.
+
+.. list-table:: Cause Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - True
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - True
+     -
+   * - YLL age group start
+     - 10 to 14 (ID=7)
+     -
+   * - YLL age group end
+     - 50 to 54 (ID=15)
+     -
+   * - YLD age group start
+     - 10 to 14 (ID=7)
+     -
+   * - YLD age group end
+     - 50 to 54 (ID=15)
+     -
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+- We are limited in that the rate of PPD in our model is informed from a systematic literature review and meta-analysis that is not location-, year-, or age-specific and the analysis of PPD incidence primarily conducted in high-resource settings.
+
+- We are limited in that we do not consider mortality associated with PPD in our model.
+
+- We are limited in that we assume all PPD cases persist for the same average duration of a single MDD episode. This is limited in the sense that duration may vary by MDD severity (for example, a longer duration for more severe cases), which could cause our estimation of YLDs to be biased. Additionally, we assume that duration of PPD is equal to the duration of all MDD episodes.
+
+- We assume that the GBD MDD severity distribution, which is based on analysis of high-resource settings, generalizes to the severity of PPD in our simulation population of interest.
+
+- We assume that the onset of PPD occurs immediately following birth. However, the onset of PPD may peak around two or three months postpartum [Shorey-et-al-2018-mncnh]_.
+
+- We assume that the incidence of PPD does not vary by pregnancy outcome (incidence is constant across partial term pregnancy loss, stillbirth, and live births)
```

Comment:
Is there any evidence about this in the Shorey et al paper?

Reply:
No, looking back at it now (this is the reference we used for the IV iron model back in the day) it's actually just among live full term births. There is some evidence that depression rates are higher among preterm infants, but the studies I have seen in my quick check are smaller among targeted populations. We could consider adding effect modification by preterm birth on depression incidence. I will add a note that we can come back to for this. I think the overall impact of adding this to the model will be small, so it may be a low priority to add.